cohortextractor/date_utils.py

```diff
@@ -50,5 +60,3 @@ def _increment_date(date, period):
                 return date.replace(day=last_day_of_month)
         else:
             return date.replace(month=1, year=date.year + 1)
-    else:
-        raise ValueError(f"Unknown time period '{period}'")
```

Comment:
This one was removed because the time period check is done earlier in the code; the assert isn't actually necessary and I only put it in in case we had other periods in future, to ensure we didn't else with anything other than "week"